### ISSUE TITLE
Abandoned project?

### ISSUE THREAD

**Issue by roukmoute** [ziadoz/awesome-php#981]
You have many pull requests and the latest updated is several months old.

**Comment by andreasschroth** [comment]
Yes, it seems abandoned, unfortunately.

**Comment by donis** [comment]
No it's not. Most of those pull requests don't conform to contributing guidelines.

**Comment by kubawerlos** [comment]
Then, it would be good to close them - 162 pending PRs does look like an abandoned project.

**Comment by rybakit** [comment]
>  Most of those pull requests don't conform to contributing guidelines.

@donis Do you mean ones marked with the "needs explanation" tag? If so, then there are "only" 35 such prs (out of 162 total opened), which is way less than "most of them".

**Comment by donis** [comment]
No, I mean PRs that have self-promotion posts, PRs that add libraries which are not "awesome" or even popular. Sure, maintainers are not adding everything they can here, but I don't see any reason to announce this abandoned just because of a high number of PRs.

**Comment by andreasschroth** [comment]
If the PRs are just self-promotion and do not conform to the contribution guidelines, they should be closed. If the PRs need further explanation / consideration / whatever they can remain open, but 162 open PRs? The oldest ones from end of 2015?

Either it's abandonned or or not taken care well of. A clean-up would do well here. That's just a personal pov and sure, it's open-source and I think we're all thankful if someone invests his spare time into this, but I think the question raised from @roukmoute is totally legit.

**Comment by ericpoe** [comment]
Once #976 is merged, it should at least help PRs that are non-compliant due to broken URLs in master. Existing PRs would need to merge master in order to get the updated non-broken URLs.

**Comment by donis** [comment]
@andreasschroth nothing "should" be done here and the maintainers maintain it as they see fit. Don't forget this is a voluntary work and no one is obliged to meet your expectations. Feel free to fork it!

@ericpoe this PR (as is #962 ) doesn't follow the rule "Please make an individual commit for each suggestion in a separate pull request."

Generally I agree, this project is very slowly updated, but there are PRs merged here and there. There is probably a good reason why maintainers are away, but how can we help to improve this situation?

**Comment by andreasschroth** [comment]
@donis I was actually thinking about forking it and yes, will probably do, but I haven't finally decided on it yet. Because if I'm gonna fork it, I want to make sure it's properly maintained over a long period if time and not just a one-time improvement. But that doesn't change anything on the impression the community here has that this project looks abandoned and I still doubt that's because of the PR quality.

**Comment by ericpoe** [comment]
@donis I guess I don't see each line as a separate suggestion but as a bug fix. "A separate suggestion" would be a separate "wow, this is awesome" suggestion. This is just to fix broken stuff.

**Comment by donis** [comment]
I definitely agree. I'm just trying to find reasons why some PRs are merged and some are not.

**Comment by ziadoz** [comment]
This project isn't abandoned. That being said, it's obvious to anyone that I've neglected it. That's primarily down to me being strict about my work/life balance, taking better care of my mental health, and things going on in my personal life which are far more important.

There are often submissions that don't meet the guidelines or are duplicates, which has affected my enthusiasm. Identifying these is most cases is manual and tedious - plus the bot that made our lives a little easier appears to be dead, which sucks. 

I'm sorry if it feels like things are quiet here, that's not my intention. Thankfully @josegonzalez does a fantastic job, so things do move along, but I'll have a think about how to bring onboard another collaborator.

@ericpoe Thanks for your clean up PR, I've merged it. There are still some outstanding errors, but I'll take a peek at them when I get a moment.

**Comment by ericpoe** [comment]
@ziadoz #988 should take care of those outstanding errors.

**Comment by ziadoz** [comment]
@ericpoe Fantastic. Thanks for sorting that, I've merged your PR.

**Comment by m1guelpf** [comment]
If you need more eyes on this, I'd be happy to help as a mantainer :)

**Comment by josegonzalez** [comment]
I've stopped really looking at this repo. I think a lot of the projects come in from people trying to promote their own stuff, and people get really upset if their issues don't get merged.

I wonder if someone could write a quick script to validate a repository against the rules of contributing...

**Comment by k4s0** [comment]
If needed, I'd be happy to help as a mantainer too :)

**Comment by andreasschroth** [comment]
Happy to help as well if wished / needed. I visit / work with Github daily and would be glad to merge PRs / reject them according to the project guidelines.

**Comment by ErikThiart** [comment]
The integration bot seems broken.

**Comment by ziadoz** [comment]
@ErikThiart The bot was indeed broken. I removed its access from this repo a few months ago though, so it won't affect new pull requests.
